src/Tests/SenseNet.Search.IntegrationTests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,20 @@
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("SenseNet.Search.IntegrationTests")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("SenseNet.Search.IntegrationTests")]
+[assembly: AssemblyCopyright("Copyright ©  2017")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
+[assembly: ComVisible(false)]
+
+[assembly: Guid("a29cb53a-df6c-487c-9fdc-dc73a5cb2b13")]
+
+// [assembly: AssemblyVersion("1.0.*")]
```

Comment:
Please fill the sn-specific values in a similar way as other assembly info files. You may copy them from other test projects. This is just so we have the same version number across all test projects.